core/notifications/src/notification_event/marketing_notification_triggered.rs

```diff
@@ -13,6 +13,7 @@ pub struct MarketingNotificationTriggered {
     pub should_add_to_history: bool,
     pub should_add_to_bulletin: bool,
     pub deep_link: Option<DeepLink>,
+    pub action: Option<Action>,
```

Comment:
@bodymindarts will this still properly deserialize payloads from the database that didn't have the action?

Reply:
yes if you add `#[serde(default)]` - then non existent `action` attributes will deserialise to `None`

Reply:
Do you add it specifically to the `action` field or to the top level `MarketingNotificationTriggered` struct?

Reply:
Just the field that was added.